_data/navigation.yml

```diff
@@ -1,16 +1,9 @@
 main:
   - title: "Services"
-    url: '/services.html'
+    url: /services.html
   - title: "Previous projects"
-    url: '/previous-projects.html'
+    url: /previous-projects.html
   - title: "About us"
-    url: '/index.html'
+    url: /index.html
   - title: "Contact us"
-    url: '#contact'
-docs:
-  - title: "List of projects"
-    children:
-      - title: "Project 1"
-        urL: '#project1'
-      - title: "Project 2"
-        urL: '#project2'
+    url: '#contact'
```

Comment:
Missing empty line as end of file

Reply:
Also some `url` variables are between '' while others don't. Please use the same format with all of them